packages/tpetra/core/src/Tpetra_MultiVector_def.hpp

```diff
@@ -4397,7 +4407,8 @@ namespace Tpetra {
     if (useHostVersion) {
       this->modify_host ();
       if (contig || isConstantStride ()) {
-        Kokkos::deep_copy (srcView_host, tgtBuf_host);
+        if (commIsInterComm)
```

Comment:
For future work, please follow Tpetra coding style; e.g., always use `{ ... }` to set off `if` statements.

Reply:
Fixed.